certified-connectors/PUG Gamified Engagement/apiProperties.json

```diff
@@ -0,0 +1,23 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "api_key": {
+        "type": "securestring",
+        "uiDefinition": {
+          "displayName": "Integration API Key",
+          "description": "Integration API Key for this API.",
+          "tooltip": "Provide your Integration API Key.",
+          "constraints": {
+            "tabIndex": 2,
+            "clearText": false,
+            "required": "true"
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#ffffff",
```

Comment:
@smart-jp White color for `iconBrandColor` is not allowed.

Documentation - [Icon](https://docs.microsoft.com/connectors/custom-connectors/certification-submission#icon)

Reply:
fixed thanks!